astroid/rebuilder.py

```diff
@@ -225,11 +226,12 @@ def _fix_doc_node_position(self, node: NodesWithDocsType) -> None:
         end_range: Optional[int] = node.doc_node.lineno
         if IMPLEMENTATION_PYPY:
             end_range = None
+        # pylint: disable-next=unsubscriptable-object
         data = "\n".join(self._data[lineno - 1 : end_range])
 
         found_start, found_end = False, False
         open_brackets = 0
-        skip_token: set[int] = {token.NEWLINE, token.INDENT}
+        skip_token: Set[int] = {token.NEWLINE, token.INDENT}
```

Comment:
I actually saw this during review and thought I had learned something new from you (`set` being able to used on < 3.8/9) but alas... 😄

Reply:
It can actually be used be used in 3.7 and 3.8 (with `from __future__ import annotations`), but not in 3.6.
Really time to remove support for 3.6 soon.

_Technically, you could use `"set[int]"` (note the quotation marks) in 3.6, but that isn't really worth it._

Reply:
Yeah, I have been looking forward to seeing you use your typing updater tool 😄